### ISSUE TITLE
Logging in with a Kitsu bot api key.

### ISSUE THREAD

**Issue by scottmcdonnell** [cgwire/gazu#318]
**Context**

Studio name: JAM Media
Gazu version: 0.10.1
Zou installation type: this is running in docker as part of the [Ayon Kitsu addon.](https://github.com/scottmcdonnell/ayon-kitsu/blob/kitsu_api_token_login/services/processor/processor/processor.py)

**Describe the bug**
I'm trying to do a login with a new Kitsu Bot and running into issues. 
I am setting the token with `gazu.setToken(api_token)` [here](https://github.com/scottmcdonnell/ayon-kitsu/blob/kitsu_api_token_login/services/processor/processor/processor.py#L129) but the call to `gazu.events.init()` [here](https://github.com/scottmcdonnell/ayon-kitsu/blob/kitsu_api_token_login/services/processor/processor/processor.py#L142) gives a error:

`ConnectionError: One or more namespaces failed to connect.' 

The same code works with a username/password.  Am I missing a step?

Strangely `gazu.client.get_api_version())` and I get version 0.19.6
but `gazu.__version__`  is 0.10.1

**Comment by scottmcdonnell** [comment]
oh dear, I figured out that my zou server had not upgraded properly and was stuck at v 0.19.6 which explains everything `{"api":"Zou","version":"0.19.6"}`

**Comment by scottmcdonnell** [comment]
upgraded zou to 0.19.12 and I get the same error.

**Comment by frankrousseau** [comment]
ping @EvanBldy

**Comment by EvanBldy** [comment]
Yeah I think the API bots are not working now with events (gazu.events.init()), I will fix that soon

**Comment by frankrousseau** [comment]
It's a complex problem to solve but we will put it in our top priority. Sorry for the late update.
